README.Rmd

```diff
@@ -85,4 +85,4 @@ monthly_prey_and_fish <- monthly_prey %>% left_join(monthly_fish)
 
 ### Dependencies
 
-The `DSMprey` package provides data for several other packages within the [CVPIA Open Science Collaborative](https://github.com/CVPIA-OSC).
+The `DSMprey` data package provides data for several other packages within the [CVPIA Open Science Collaborative](https://github.com/CVPIA-OSC).
```

Comment:
The DSMprey package is not providing data for these packages. It provides data for the bioenergetics model. Would be good to clarify that here.

Reply:
Also correct that in the paragraph above "All data can be used in the CVPIA salmon life cycle models."